exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/partitionsender/Partitioner.java

```diff
@@ -29,6 +29,8 @@
 import org.apache.drill.exec.record.RecordBatch;
 
 public interface Partitioner {
+  int DEFAULT_RECORD_BATCH_SIZE = (1 << 10) - 1;
```

Comment:
The DEFAULT_RECORD_BATCH_SIZE is already in the current master branch here [1].  I simply moved it into the `Partitioner` interface. 

[1] https://github.com/apache/drill/blob/master/exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/partitionsender/PartitionerTemplate.java#L58

Reply:
I see, thanks for clarification.
Is that comment regarding `recordCount` isn't necessary anymore?

Reply:
I think that was an omission.  I have re-inserted the comment but with some modifications because the batch-sizing project now allows operators to set the output batch size in terms of Mbytes rather than `recordCount`.  It is not yet applied across the board, so this `DEFAULT_RECORD_BATCH_SIZE` is still relevant for the exchange operators.